src/common.mak

```diff
@@ -0,0 +1,204 @@
+# Common Makefile variables (shared between DMD, DRuntime and Phobos)
+# Be careful when updating
+# This file expects the following variables to be defined in the including Makefile:
+# - D_HOME        Root of all D repositories
+
+###############################################################################
+# Common directories
+################################################################################
+
+DMD_DIR = $(D_HOME)/dmd
+DRUNTIME_DIR = $(D_HOME)/druntime
+PHOBOS_DIR = $(D_HOME)/phobos
+DLANG_ORG_DIR = $(D_HOME)/dlang.org
+TOOLS_DIR = $(D_HOME)/tools
+INSTALL_DIR = $(D_HOME)/install
+INSTALLER_DIR = $(D_HOME)/installer
+TMP?=/tmp
```

Comment:
Here the convention changes from "spaces around =" to "no spaces around ="

Reply:
> Here the convention changes from "spaces around =" to "no spaces around ="

Ah yes, this was due to the definitions united from multiple Makefiles with no consistent style.